compair/static/modules/assignment/assignment-module_spec.js

```diff
@@ -1207,6 +1207,22 @@ describe('assignment-module', function () {
                     $rootScope.assignment = angular.copy(mockAssignment);
                     $rootScope.assignment.id = undefined;
                     $rootScope.assignment.availableCheck = true;
+                    if ($rootScope.date.astart.date == null) {
```

Comment:
suggest you just set the value instead of using if conditions in tests (the if should always pass/fail since the state should be known and consistent so the if isn't really needed)